README.md

```diff
@@ -534,6 +534,24 @@ Related Fedora ticket: [Bug 524727 - Dracut + encrypted root + networking (2009)
 - [openSUSE](https://build.opensuse.org/package/show/openSUSE:Factory/dracut-sshd)
 - [Arch AUR](https://aur.archlinux.org/packages/dracut-sshd-git)
 
+### Build Debian/Ubuntu package
+
+This repository include necessary code to build your own .deb package.
+Here are the steps to follow:
+
+1. (Optional) In case you do not want to install Debian development
+   packages in your host system, you may choose to use Docker. Assuming
+   you have installed Docker on your system, launch a Docker container
+   with `docker run -i -t -w /root debian:stable-slim`
+2. Install `git-buildpackage`: `apt-get update && apt-get install git-buildpackage`
+3. Clone the repository: `gbp clone https://github.com/gsauthof/dracut-sshd.git`
+4. Enter the folder: `cd dracut-sshd`
+5. Build the package: `gbp buildpackage --git-upstream-tree=branch --no-sign`
+6. The debian packages and relavant files should be in the parent folder.
+
+For more information regarding proper Debian packaging, please refer to
+[Debian wiki page for packaging with Git][debian-package-git] and
+[Git-buildpackage official website][git-buildpackage].
```

Comment:
I prefer to  move this to a small README.md in that debian subdirectory.

It feels like this debian packaging intro is only remotely related to the topic of the main README which is already long enough.

Reply:
Moved to debian/README.md